dss-signature-policy-validator/src/main/java/eu/europa/esig/dss/signature/policy/validation/FullCAdESSignaturePolicyValidator.java

```diff
@@ -79,12 +81,20 @@
 	private Set<CertificateToken> signerCertPath = null;
 
 	private SignaturePolicy asn1SignaturePolicy;
+	
+	private ResourceBundle bundle;
+	
+	private static final String BUNDLE = "fullcadesvalidationmessages";
+	
+	
 
 	public FullCAdESSignaturePolicyValidator() {
+		bundle = ResourceBundle.getBundle(BUNDLE);
```

Comment:
Having in mind that there is no way to override locale, how will the user resources be localized? Any ideas?

Reply:
We have to decide what will be done when we implement localization in DSS, not only for this validation, but for the other messages from core DSS validation. 

If we will not change code from other modules, and becomes not practical to set the Locale in the validators, the best option probably would set the codes instead of the formatted messages, and make a post-processor that will localize the messages.